src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -139,7 +133,7 @@ private String decrypt(final String key, final String value) {
         try {
             return getStorageHelper().decrypt(value);
         } catch (GeneralSecurityException | IOException e) {
-            Logger.e(TAG, "Decryption failure", "", ADALError.ENCRYPTION_FAILED, e);
+            Logger.e(TAG, "Decryption failure", "", ADALError.DECRYPTION_FAILED, e);
```

Comment:
Does ClientAnalytics record the calling app details such as name, version etc?

---
In reply to: [60865909](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/607#discussion_r60865909) [](ancestors = 60865909)